docs/reference/css_classes.md

```diff
@@ -55,14 +55,26 @@ Construct a CSS class attribute by joining together the controller identifier an
 
 **Note:** CSS class attributes must be specified on the same element as the `data-controller` attribute.
 
+<meta data-controller="callout" data-callout-text-value="data-search-loading-class=&quot;bg-gray-500 animate-spinner cursor-busy&quot;">
+
+```html
+<form data-controller="search"
+      data-search-loading-class="bg-gray-500 animate-spinner cursor-busy">
+  <input data-action="search#loadResults">
+</form>
+```
+
+If you are working with utility classes and want to define multiple CSS classes for an attribute, separate the CSS class names with spaces.
```

Comment:
Let's drop "utility classes" from the docs since it's not the *only* reason for this change and is more of a convention. Fine to just frame it around "multiple classes".